packages/react-components/src/localization/locales/fr-FR/strings.json

```diff
@@ -2,7 +2,9 @@
   "participantItem": {
     "isMeText": "(vous)",
     "menuTitle": "Plus d’options",
-    "removeButtonLabel": "Supprimer"
+    "removeButtonLabel": "Supprimer",
+    "sharingIconLabel": "Partage",
+    "mutedIconLabel": "Muet activé"

```

Comment:
why does it sound so wrong :D
Might be me not talking French so often or at least not using French language in software/browser...